roboteam_ai/src/conditions/CanPlay.cpp

```diff
@@ -4,23 +4,23 @@
  * USED FOR CHECKING GAME STATE CHANGES DONT USE IF YOU DONT KNOW WHEN TO
  */
 
+#include <roboteam_ai/src/interface/api/Output.h>
 #include "../utilities/Constants.h"
 #include "CanPlay.h"
 
 namespace rtt {
 namespace ai {
 
 CanPlay::CanPlay(std::string name, bt::Blackboard::Ptr blackboard)
-        :Condition(std::move(name), std::move(blackboard)) { };
-
+        :Condition(std::move(name), std::move(blackboard)) {
+};
 
 bt::Node::Status CanPlay::onUpdate() {
-    Vector2 ballVel=ball->vel;
-
-    bool ballIsLayingStill = ballVel.length() < Constants::BALL_STILL_VEL();
-    auto refCommand = static_cast<RefGameState>(rtt::ai::Referee::getRefereeData().command.command);
+    bool ballIsLayingStill = GameStateManager::getCurrentGameState().ballPositionAtStartOfGameState.dist(ball->pos) < 0.05;
+    auto refCommand = static_cast<RefCommand>(rtt::ai::GameStateManager::getRefereeData().command.command);
```

Comment:
That’s the way to do it though... it should be safe because the integer is known. Getting an enumeration from ROS seems unlikely to me but we can always look that up.

Reply:
https://stackoverflow.com/questions/8357240/how-to-automatically-convert-strongly-typed-enum-into-int

Reply:
hmm ok.